README.rst

```diff
@@ -145,10 +136,16 @@ The project is currently maintained by a team of volunteers.
 
 **Note**: `scikit-learn` was previously referred to as `scikits.learn`.
 
-
 Help and Support
 ----------------
 
+Submitting a Pull Request
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Before opening a Pull Request, have a look at the
+full Contributing page to make sure your code complies
+with our guidelines: http://scikit-learn.org/stable/developers/index.html
+
```

Comment:
Not sure we should change this. It makes more sense under "development" like before, than under "Help and Support"

Reply:
I can revert. Just thought having supportive activities toward the end could be a good focal area for users.

Reply:
Reverted 'Submitting a Pull Request' position